samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,126 @@
+// <copyright file="app-cache-tab.tsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useState } from "react";
+import React from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/// </summary>
+/// function logitem
+/// </summary>
+function logItem(action: string, actionColor: string, message: string) {
+  const newItem =
+    "<span style='font-weight:bold;color:" +
+    actionColor +
+    "'>" +
+    action +
+    "</span> " +
+    message +
+    "</br>";
+  return newItem;
+}
+
+/// </summary>
+/// In beforeUnloadHandler using setItems and readyToUnload callback function
+/// </summary>
+const beforeUnloadHandler = (
+  setItems: React.Dispatch<React.SetStateAction<string[]>>,
+  readyToUnload: () => void) => {
+
+  let newItem = logItem("OnBeforeUnload", "purple", "Started");
+  setItems((Items) => [...Items, newItem]);
+
+  newItem = logItem("OnBeforeUnload", "purple", "Completed");
```

Comment:
the started/completed serve no purpose here. just load onBeforeUnload received

Reply:
comment fixed